### PR TITLE
Upgrade google-api-client, fix broken tests

### PR DESCRIPTION
**Pull request by icco**
This takes #468 and tries to fix the failing tests as well.

### CODE REVIEW

**Comment by Temikus** [comment]
Please do not merge yet as this is actually worrying as empty get should return all zones. Is this only failing with the new lib version?

**Comment by Temikus** [comment]
Ah sorry nvm read the diff wrong m(_ _)m

**Comment by Temikus** [comment]
Ah sorry nvm read the diff wrong m(_ _)m

**Comment by icco** [comment]
So it's failing with some subset of zones, which is concerning. Also some stackdriver tests are failing, which I haven't been able to figure out yet.

**Comment by icco** [comment]
Hmm, just seems to be flaky. Added some debugging, and the tests stopped failing... whee.

File: test/integration/monitoring/test_timeseries.rb [hunk 1]
```diff
@@ -151,6 +151,9 @@ def test_multiple_timeseries
         :page_size => 1
       )
     end
+
+    mu_pp_for_diff(resp)
```
Comment (Temikus) [inline test/integration/monitoring/test_timeseries.rb]: Do you intentionally want to leave this here or is it a leftover from debugging?

Comment (icco) [reply test/integration/monitoring/test_timeseries.rb]: Leftover. Will remove.

**Pull request merged**